platform/ios/src/MGLMapView.mm

```diff
@@ -87,6 +88,10 @@
 const CGFloat MGLMapViewDecelerationRateFast = UIScrollViewDecelerationRateFast;
 const CGFloat MGLMapViewDecelerationRateImmediate = 0.0;
 
+const MGLMapViewPreferredFramesPerSecond MGLMapViewPreferredFramesPerSecondDefault = -1;
+const MGLMapViewPreferredFramesPerSecond MGLMapViewPreferredFramesPerSecondLowPower = 30;
+const MGLMapViewPreferredFramesPerSecond MGLMapViewPreferredFramesPerSecondMaximum = 60;
```

Comment:
Should `.lowPower` and `.maximum` also be based per-device type?

Reply:
I think there’s room for that kind of functionality, yeah — testing on an iPhone 5, it does seem like 15 FPS may be more appropriate for that device.